src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx

```diff
@@ -281,7 +281,12 @@ export default function WalletBreakdownView({
       .dividedBy(new BigNumber(totalFiatValue))
       .multipliedBy(100)
       .toFixed(2);
-
+  console.debug(
+    'chartBalances',
+    chartBalances.length,
+    'breakdownBalances: ',
+    breakdownBalances.length
+  );
```

Comment:
Do we need to keep this ?
If so prefer `logger` from `@utils`

Reply:
Didn't see this 😆